h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -36,7 +39,7 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
             @annotator.triggerHideFrame()
     ,
       "title": "Hide Highlights"
-      "class": "h-icon-visibility"
+      "class": "annotator-highlight-visibility h-icon-visibility"
```

Comment:
That might mean adding another key to each of these button definitions that you use to set the data attribute in `makeButton`.